wiremock/mappings/X778160-full.json

```diff
@@ -163,9 +163,8 @@
         "jsonBody": {
```

Comment:
Can we remove the .DS_Store file that has been checked in?

Reply:
Resolved